src/iOS/toga_iOS/widgets/scrollcontainer.py

```diff
@@ -19,86 +17,110 @@
 
 
 class ScrollContainer(Widget):
-    __current_content = None
-    __scroll_height_constraint = None
-    __scroll_width_constraint = None
-    __vertical_enabled = False
-    __horizontal_enabled = False
-    
-    def __update_content_size(self):
+    def update_content_size(self):
         # We need a layout pass to figure out how big the scrollable area should be
-        self.__current_content.interface.refresh()
+        self.current_content.interface.refresh()
```

Comment:
There's no need to call this "current" content - there's no storage of "historical" content, so "current" content just makes the variable name longer.

Reply:
I've removed this variable, it serves the same purpose as self.interface.content